.github/workflows/docker.yml

```diff
@@ -28,6 +28,10 @@ jobs:
 
       - name: Inspect builder
         run: docker buildx inspect
+      
+      - name: Extract version from pyproject.toml
+        run: |
+          echo "SYNAPSE_VERSION=$(grep "^version" pyproject.toml | sed -E 's/version\s*=\s*["]([^"]*)["]/\1/')" >> $GITHUB_ENV
```

Comment:
When I test this locally I see:

```
$ echo "SYNAPSE_VERSION=$(grep "^version" pyproject.toml | sed -E 's/version\s*=\s*["]([^"]*)["]/\1/')"
SYNAPSE_VERSION=version = "1.88.0"
```

which does not look correct to me.

Reply:
Ahh, this is because I have `grep` aliased:

```
$ type grep
grep is aliased to `grep --color=always'
```

Bypassing the alias fixes shows that it works:

```
$ echo "SYNAPSE_VERSION=$(\grep "^version" pyproject.toml | sed -E 's/version\s*=\s*["]([^"]*)["]/\1/')"
SYNAPSE_VERSION=1.88.0
```